CONTRIBUTING.md

```diff
@@ -38,7 +45,7 @@ Note that the merge commit to which these tags are attached represents an entire
 Each time a merge commit is made to the master branch of the `protocols` repo, a 'mirror read-only' repository (protocols-website) will be automatically triggered to build the rendered html versions of the protocols using GitHub Actions.
 The resulting website is hosted at **TO BE ADDED**. This website will host all approved and published versions of all protocols.
 
-## Workflow {#workflow}
+## <a name="workflow"></a>Workflow
```

Comment:
Just FYI this should normally not be needed, as the name attribute equals the title (as lower-case).

Reply:
Fixed in 0cb018b